packages/kbn-es/src/artifact.js

```diff
@@ -284,6 +284,8 @@ exports.Artifact = class Artifact {
       new Transform({
         transform(chunk, encoding, cb) {
           contentLength += Buffer.byteLength(chunk);
+          const downloadedMB = Math.round(contentLength / (1024 * 1024));
+          process.stdout.write(`Downloaded ${downloadedMB} MB\r`);
```

Comment:
This should use the `this._log` to write the progress information, and can we either log with `verbose` level or debounce this information a bit?